tests/templates/kuttl/smoke/10-assert.yaml.j2

```diff
@@ -404,7 +403,6 @@ data:
     network.host: "0.0.0.0"
     node.store.allow_mmap: "false"
     plugins.security.allow_default_init_securityindex: "true"
-    plugins.security.authcz.admin_dn: "CN=generated certificate for pod"
```

Comment:
Is this no longer needed because of https://github.com/stackabletech/opensearch-operator/pull/25?

Reply:
It is not needed in our tests and can therefore be removed.

It was necessary when the security plugin was initialized by the `securityadmin.sh` script. But the security plugin is now initialized with `plugins.security.allow_default_init_securityindex: "true"`.

The `securityadmin.sh` script will be required again to update the configuration. But we have to decide then how the authorization will work and add the according configuration property via the operator.